templates/test.template

```diff
@@ -2,21 +2,20 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-template: default
 author: Mathias Bynens
-desc: Unicode property escapes `\p{<%= UnicodePropertyValueExpression %>}` and `\P{<%= UnicodePropertyValueExpression %>}`
+description: >
+  Unicode property escapes `\p{}` and \P{}` for `<%= UnicodePropertyValueExpression %>`
```

Comment:
I also shortened the description here. It's easier - for me - to read. If it's wrong, I can revert this.